noggin/templates/user-settings-otp.html

```diff
@@ -71,7 +71,7 @@ <h5 id="pageheading">{{ _("OTP Tokens") }}</h5>
   <div class="list-group-item {{'text-muted bg-light' if token.disabled}}">
     <div class="row align-items-center">
       <div class="col h6">
-        <div data-role="token-description">{{token.description}}</div>
+        <div data-role="token-description">{{token.description if token.description}}</div>
```

Comment:
It would probably be better to wrap the whole tag in `{% if token.description %} ... {% endif %}`, no? This way it won't leave an empty tag, unless you need it for layout purposes?

Reply:
Ah, maybe you use that for the unit test.